packages/realm-app-importer/src/AppImporter.ts

```diff
@@ -233,9 +233,10 @@ export class AppImporter {
     }
   }
 
-  private loadAppConfigJson(appTemplatePath: string): AppConfig {
+  private loadAppConfigJson(appTemplatePath: string, replacements: TemplateReplacements = {}): AppConfig {
```

Comment:
Perhaps this should take the `appName` as a (optional) parameter instead?

Reply:
Yeah, we can try that.  The underlying code now looks like this:
```typescript
  private loadAppConfigJson(appTemplatePath: string, appNameOverride?: string): AppConfig {
    const configJsonPath = path.resolve(appTemplatePath, "config.json");
    const configJson = this.loadJson(configJsonPath);
    return appNameOverride ? { ...configJson, ...{ name: appNameOverride } } : configJson;
  }
```

Reply:
and now:
```typscript
    const appNameOverride = { ...replacements["config.json"] }?.name as string;
 ```

Reply:
Kind of like the other way better, as we are "loading" `AppConfigJson`, which could possibly include other template replacements other than `name`